v0/destinations/user/data/USERGroupConfig.json

```diff
@@ -0,0 +1,76 @@
+[
+  {
+    "sourceKeys": "userId",
+    "destKey": "user_custom_id",
+    "required": true,
+    "sourceFromGenericMap": true
+  },
+  {
+    "sourceKeys": "groupId",
+    "destKey": "custom_id",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "sourceKeys": "traits.name",
+    "destKey": "name",
```

Comment:
Why we have not included the context.traits.name ? Can you please clearify?

Reply:
because for group call root level traits contain the group parameters only. Not the context one